python/grass/jupyter/display.py

```diff
@@ -53,25 +53,40 @@ def run(self, module, **kwargs):
         else:
             raise ValueError("Module must begin with letter 'd'.")
 
-    def __getattr__(self, attr, **kwargs):
+    def __getattr__(self, name, **kwargs):
         """Parse attribute to GRASS display module. Attribute should be in
-        the form 'd_{module}'."""
+        the form 'd_{module}'.
 
-        # First check to make sure format is correct
-        if attr[:2] != "d_":
-            print(attr[:2])
-            raise ValueError("Module must begin with 'd_'.")
-        # Now reformat string
-        grass_module = f"d.{attr[2:]}"
+        :param str name: display module in form 'd_{module}'
+        :param kwargs: arguments to be passed to display module
+
+        :return: output of display module
+
+        ----- Example -----
+        # Create map instance
+        >>> m = GrassRenderer()
+        # Add elevation raster with shortcut
+        >>> m.d_rast(map="elevation")
+        # Add legend with shortcut
+        >>> m.d_legend(raster="elevation")
+        # Show map
+        >>> m.show()
+        -------------------
+        """
+
+        # Check to make sure format is correct
+        if not name.startswith("d_"):
+            raise AttributeError("Module must begin with 'd_'.")
```

Comment:
Python exception messages don't end with `.`, test e.g.:

```
>>> xxx
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'xxx' is not defined
>>> def a():
...     pass
... 
>>> a(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a() takes 0 positional arguments but 1 was given
```

I think exceptions raised in GRASS code are translatable (check the other code) which in GRASS means enclosing the string in `_(...)` (call to a function named `_` which is defined as a buildin and returns the translated message if locale language is not English and the message was translated).